test/test_json_requests.py

```diff
@@ -0,0 +1,34 @@
+import requests
+from mockserver import request, response, times, request_json
+from test import MOCK_SERVER_URL, MockServerClientTestCase
+
+
+class TestJsonExpectations(MockServerClientTestCase):
+
+    def test_expect_json_body(self):
+        self.client.expect(
+            request(body=request_json({"key": "value", "key1": "value2"})),
+            response(),
+            times(1)
+        )
+
+        result = requests.get(MOCK_SERVER_URL + "/path")
+        self.assertEquals(result.status_code, 404)
+
+        result = requests.get(MOCK_SERVER_URL + "/path", json={"key": "value", "key1": "value2"})
+        self.assertEquals(result.status_code, 200)
+
+    def test_expect_json_body_order(self):
```

Comment:
i don't see anything different in this test

you only need 2 tests here, one to demonstrate the behavior of strict_json and one for json_fields or whichever name it is